templates/experiments/single_experiment_home.html

```diff
@@ -375,19 +374,21 @@ <h3 class="font-bold text-lg">Parent Experiments:
                                 type="text"
                                 x-model="filter.value"
                                 @input.debounce="triggerFilterChange()"
-                                class="w-40 input input-bordered input-sm"
+                                class="w-40 block px-3 border border-base-200 rounded-lg bg-base-100 text-sm"
+                                style="height: 2rem;"
                                 placeholder="Enter value..."
                               >
                             </template>
                           <!-- Timestamp Input -->
                             <template x-if="filterData.columns[filter.column]?.type === 'timestamp'">
-                              <div class="relative">
+                              <div class="relative" x-init="if (!('isValidDate' in filter)) filter.isValidDate = true">
                                 <input
                                   type="date"
                                   x-model="filter.value"
                                   @input="filter.isValidDate = $event.target.validity.valid"
                                   @change="filter.isValidDate && triggerFilterChange()"
-                                  class="input input-bordered input-sm w-40"
+                                  class="w-40 block px-3 border border-base-200 rounded-lg bg-base-100 text-sm"
+                                  style="height: 2rem;"
```

Comment:
This is equivalent to using the `h-8` class

Reply:
I tried add the `h-8` class, but it doesn't apply unless I explicitly put it in the style, so I'm not quite sure where it's getting overridden

Reply:
Did you run `npm run dev` and reload the page without cache?

Not a big issue - fine to leave it if you can't make it work.

Reply:
yep tried both of those and no luck